ecommerce/admin_test.py

```diff
@@ -31,3 +31,14 @@ def test_save_and_log_model(self):
             change=Mock()
         )
         assert OrderAudit.objects.count() == 1
+
+        line = LineFactory.create()
```

Comment:
@pdpinch i think here we should separate test, that should create a Line with a dummy course key, then when should assert `OrderAudit` change. Afterword we should update course key in same Line and see the behaviour in  `OrderAudit`